scss/tech.scss

```diff
@@ -0,0 +1,44 @@
+.tech {
+    background: url('./assets/react_icon.svg') no-repeat;
```

Comment:
The indentation should be of 2 spaces. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation). And why as background? In this case you should use `img` tag